box/info.go

```diff
@@ -0,0 +1,100 @@
+package box
+
+import (
+	"fmt"
+
+	"github.com/tarantool/go-tarantool/v2"
+	"github.com/vmihailenco/msgpack/v5"
+)
+
+var _ tarantool.Request = (*InfoRequest)(nil)
+
+// ClusterInfo represents information about the cluster.
+// It contains the unique identifier (UUID) of the cluster.
+type ClusterInfo struct {
+	UUID string `msgpack:"uuid"`
+}
+
+// Info represents detailed information about the Tarantool instance.
+// It includes version, node ID, read-only status, process ID, cluster information, and more.
+type Info struct {
+	// The Version of the Tarantool instance.
+	Version string `msgpack:"version"`
+	// The node ID (nullable).
+	ID *int `msgpack:"id"`
```

Comment:
If I'm not mistaken replicaset id starts from 1, so 0 is a valid-nil value. If I'm wrong, there's no need to fix it.
```suggestion
	// The node ID (nullable).
	ID int `msgpack:"id"`
```

Reply:
Still actual.

Reply:
```
tarantool> box.info
---
- version: 2.11.6-entrypoint-4-g5a3c21534e
  id: null
```
if box.cfg has not yet been called - may be null